src/renderer/initializers/kube-object-detail-registry.tsx

```diff
@@ -54,7 +54,7 @@ import { ReplicaSetDetails } from "../components/+workloads-replicasets";
 import { StatefulSetDetails } from "../components/+workloads-statefulsets";
 import type { KubeObjectDetailsProps } from "../components/kube-object-details";
 
-export function intiKubeObjectDetailRegistry() {
+export function initKubeObjectDetailRegistry() {
```

Comment:
Good catch 👍  Although this could/should be in separate PR, since it's out of scope of this PR and probably would land to `master` faster.

Reply:
Well, true. I don't think that it's needed here. :) It will get there eventually.

Reply:
Hmm. It could just be better to keep the overhead for trivial cleanup / boy scouting at minimum. It won't strain the PR if a reviewer simply skips/skims commits that are clearly trivial. And when it comes to clean-up, I would choose quantity over fast-tracking to ´master´.